src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -468,12 +468,15 @@ internal override void ForceComplete(SourceLocation locationOpt, CancellationTok
                         break;
 
                     case CompletionPart.EnumUnderlyingType:
-                        var discarded = this.EnumUnderlyingType;
+                        {
+                            _ = this.EnumUnderlyingType;
+                        }
```

Comment:
wat...

Reply:
Need to adjust the fixer here to special case switch sections...

Reply:
There should be a helper in syntax facts: either IsExecutableBlock or IsPureBlock depending on your needs.

Reply:
Never mind, I think that's in one of my PRs that's not merged yet.